packages/common/container-definitions/api-report/container-definitions.alpha.api.md

```diff
@@ -330,8 +330,6 @@ export interface IFluidPackageEnvironment {
     };
 }
 
-export { IGenericError }
```

Comment:
Just for visibility, these are `@internal` items in another package that were re-exported by this package. Semantically, they shouldn't be re-exported in any roll-up except `@internal` ones.

We _could_ manage this a bit better by leveraging API-Extractor's [bundledPackages](https://api-extractor.com/pages/configs/api-extractor_json/#bundledpackages) feature globally, but this does incur performance costs, and would need to be validated.

Reply:
Changes to remove the funkiness: #21602